### DISCUSSION TITLE
Avoid too many Blank Lines

### DISCUSSION THREAD

**Discussion by christianbraeunlich** [StefanMaron/BusinessCentral.LinterCop#174] [New Rule]
I'd like to avoid too many blank lines.

Inspired by:
- https://www.python.org/dev/peps/pep-0008/#blank-lines
- https://www.flake8rules.com/rules/E303.html

**Comment by fvet** [comment]
Not a rule, but definitely interesting if this could be part of the 'Code Cleanup' actions.
https://github.com/anzwdev/al-code-outline/issues/347

**Reply by christianbraeunlich** [reply]
If there is an action for it, then there is a rule for it. The action is based on this rule...or is a rule based on an action? If for some reason you don't apply the action, you run the risk of committing those unnecessary blank lines. A centralized pipeline ensures that this rule is reviewed.
